api/build.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
```

Comment:
remove.  for springboot you don't need build war -Pls build jar. Use embedded Tomcat